css/style.css

```diff
@@ -2070,6 +2086,10 @@
 }
 
 @media (min-width: 375px) and (max-width: 399px) {
+  .tesla-car {
+      height: 175px;
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Tesla-pro/pullRequest?prid=6145016)